src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqModelRepository.kt

```diff
@@ -6,40 +6,40 @@ import org.vaccineimpact.api.app.repositories.ModelRepository
 import org.vaccineimpact.api.db.JooqContext
 import org.vaccineimpact.api.db.Tables.*
 import org.vaccineimpact.api.db.fromJoinPath
+import org.vaccineimpact.api.db.joinPath
+import org.vaccineimpact.api.db.tables.records.ModelVersionRecord
+import org.vaccineimpact.api.models.Country
 import org.vaccineimpact.api.models.Model
 import org.vaccineimpact.api.models.ModelVersion
 
 class JooqModelRepository(dsl: DSLContext) : JooqRepository(dsl), ModelRepository
 {
-
-    private fun modelQuery(): SelectConditionStep<*>
-    {
-        return dsl.select(
-                MODEL.ID,
-                MODEL.DESCRIPTION,
-                MODEL.CITATION,
-                MODEL.MODELLING_GROUP,
-                MODEL.GENDER_SPECIFIC,
-                GENDER.CODE.`as`("gender"),
-                MODEL.CURRENT_VERSION)
-                .fromJoinPath(MODEL, GENDER, joinType = JoinType.LEFT_OUTER_JOIN)
-                .where(MODEL.IS_CURRENT.eq(true))
-    }
-
     override fun all(): List<Model>
     {
         val modelRecords =  modelQuery()
 
-        val currentVersions = dsl.select(*MODEL_VERSION.fields())
+        val currentVersionRecords = dsl.select(*MODEL_VERSION.fields(),*COUNTRY.fields())
                 .from(MODEL_VERSION)
                 .innerJoin(MODEL)
                 .on(MODEL_VERSION.ID.eq(MODEL.CURRENT_VERSION))
-                .fetch()
-                .associate{ it[MODEL_VERSION.ID] to it.into(ModelVersion::class.java) }
+                .joinPath(MODEL_VERSION, MODEL_VERSION_COUNTRY, COUNTRY, joinType=JoinType.LEFT_OUTER_JOIN)
+
+        val currentVersions = currentVersionRecords.groupBy{it[MODEL_VERSION.ID]}
+                .mapValues{
+                    val countries =  it.value.mapNotNull{r ->
```

Comment:
Just a couple of notes on formatting - could we stick with explicit braces for if statements (here and below) and could you autoformat the code (ctrl+alt+l) as the spacing is a bit off?